hack/install-tools.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+# helper script to install build auxiliary tools in local directory
+#
+# usage:
+#   GOBIN=<dir> [TMPDIR=<tmpdir>] install-tools.sh --<tool-name>
+#
+set -e
+GO_CMD=${GO_CMD:-$(command -v go)}
+GOBIN=${GOBIN:-$(pwd)/bin}
+TMPDIR=${TMPDIR:-$(pwd)/tmp}
```

Comment:
Let's use `mktmp -d` in the functions that need a temporary dir.

Reply:
ACK